### DISCUSSION TITLE
Iw3 crashing when using -vf v360 filter

### DISCUSSION THREAD

**Discussion by MaryAtlas** [nagadomi/nunif#229] [Q&A] [answered]
Hello.
When I use any v360 filter on -vf(src), for example, I typed v360=flat:fisheye on the -vf(src) box, and click Start, it will show Loading "Depth Model" for 20 seconds, then iw3 will auto close itself. It runs without the v360 filter. I looked at my task manager and saw Python suspended itself, following the iw3 crash. Any idea on a fix?

![Capture](https://github.com/user-attachments/assets/4125e7d4-5708-4c3f-8013-05e3427c32c8)

**Comment by nagadomi** [comment, marked as answer]
Fixed https://github.com/nagadomi/nunif/commit/ec642e406f2260da38794b89c15bcbb3d853bd3c. Run `update.bat` and try again.

Also, v360 filter will not work because the depth estimation model and stereo generation model do not expect FlashEye or Equirectangular image input.
I recommend simply using video player that supports Full SBS.

**Reply by MaryAtlas** [reply]
Thank you very much for the fix update. It fixed it. 
I am not exactly trying to completely make a Fisheye video, but trying to change the proportion to fit a certain format of my video player. I tried a small snippet of my video just now and the output looks like it is working as I envisioned, but I have not yet tried out the full thing that I am thinking about. Your fix will not be in vain, and is very helpful for me currently.
